### ISSUE TITLE
Not enough space

### ISSUE THREAD

**Issue by terminal-cs** [waydroid/waydroid#822]
**Describe the bug**
There is not enough space for 'war robots', a game.

**Screenshots**
![image](https://user-images.githubusercontent.com/76945439/226725702-0c896aae-cc79-48c0-ad09-aa59b4023006.png)

**General information (please complete the following information):**
 - not applicable

**Desktop (please complete the following information):**
 - not applicable

**Smartphone (please complete the following information):**
 - not applicable

**Additional context**
It shows i have 10 gb still free but aparently it needs more? I don't know how to extend it

**Logs (please upload as file)**
 - Prop file, located at `/var/lib/waydroid/waydroid.prop`. In case that doesn't exist, use `/var/lib/waydroid/waydroid_base.prop` instead
 - not applicable
 - Log file, located at `/var/lib/waydroid/waydroid.log`
 - not applicable
 - *Full* (not partial) output of `waydroid logcat` while Waydroid is running
 - not applicable
 - Output of `dmesg`
 - not applicable

**Comment by lordmemed** [comment]
have you tried this?

>Some of these games have data/obb permission issues, which can be resolved by manually setting permissions to 777.
`chmod 777 -R /sdcard/Android`
`chmod 777 -R /data/media/0/Android `
`chmod 777 -R /sdcard/Android/data`
`chmod 777 -R /data/media/0/Android/obb `
`chmod 777 -R /mnt/*/*/*/*/Android/data`
`chmod 777 -R /mnt/*/*/*/*/Android/obb`

for me either that or my disk just fail and fallback to read-only mode :D

**Comment by terminal-cs** [comment]
> have you tried this?
> 
> > Some of these games have data/obb permission issues, which can be resolved by manually setting permissions to 777.
> > `chmod 777 -R /sdcard/Android`
> > `chmod 777 -R /data/media/0/Android `
> > `chmod 777 -R /sdcard/Android/data`
> > `chmod 777 -R /data/media/0/Android/obb `
> > `chmod 777 -R /mnt/*/*/*/*/Android/data`
> > `chmod 777 -R /mnt/*/*/*/*/Android/obb`
> 
> for me either that or my disk just fail and fallback to read-only mode :D

Should i run this in a waydroid adb shell?

**Comment by lordmemed** [comment]
> Should i run this in a waydroid adb shell?

no, run this in waydroid shell (or rooted android, terminal app), open terminal then type `sudo waydroid shell`

**Comment by terminal-cs** [comment]
> > Should i run this in a waydroid adb shell?
> 
> no, run this in waydroid shell (or rooted android, terminal app), open terminal then type `sudo waydroid shell`

That's what I was talking about and did, it didn't work. The game still says there's not enough space

**Comment by lordmemed** [comment]
> > > Should i run this in a waydroid adb shell?
> > 
> > 
> > no, run this in waydroid shell (or rooted android, terminal app), open terminal then type `sudo waydroid shell`
> 
> That's what I was talking about and did, it didn't work. The game still says there's not enough space

ouh, sorry about that.

it's quite sad, but when i encounter this problem and still can't run the app/game because of storage issue, i reset waydroid.
maybe this problem not encountered by many, just some of us.

**Comment by Luki3532** [comment]
I am having the same issue

**Comment by linsyking** [comment]
I also encountered this issue and strangely `/sdcard` is an empty folder. I suspect the problem is permission problem as I am using a network filesystem.
